onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -774,14 +774,13 @@ common::Status TensorrtExecutionProvider::Provider_Compile(const std::vector<onn
     if (!has_dynamic_shape) {
       std::string trt_node_name_with_precision_shape = trt_node_name_with_precision + "_" + GetVecHash(input_shapes);
       std::string cached_path = GetEnginePath(engine_cache_path_, trt_node_name_with_precision_shape);
-      std::ifstream planFile(cached_path, std::ios::binary | std::ios::in);
-      if (planFile && engine_cache_enable_) {
-        planFile.seekg(0, std::ios::end);
-        int engine_size = planFile.tellg();
-        planFile.seekg(0, std::ios::beg);
+      std::ifstream plain_file(cached_path, std::ios::binary | std::ios::in);
```

Comment:
we can fix the naming convention to _
but please rename it back to plan_file  , which has specific meaning in TensorRT
reference https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html
and search for "plan file"